include/rfb.js

```diff
@@ -1131,6 +1138,19 @@ var RFB;
                 case 3:  // ServerCutText
                     return this._handle_server_cut_text();
 
+                case 150: // EndOfContinuousUpdates
+                    var first = !(this._supportsContinuousUpdates);
+                    this._supportsContinuousUpdates = true;
+                    this._enabledContinuousUpdates = false;
+                    if (first) {
+                        this._enabledContinuousUpdates = true;
+                        this._updateContinuousUpdates();
+                        Util.Info("Enabling continuous updates.");
+                    } else {
+                        // FIXME: may need to send a framebufferupdaterequest here
```

Comment:
Yeah, probably, although we should only receive this message if we ask to stop the requests, right?
Still, just in case, it's probably not a bad idea.